src/notifications/datastore/notifications.js

```diff
@@ -43,14 +43,14 @@ export default {
     },
     enrichContext: (state, getters, rootState, rootGetters) => context => {
       if (!context) return
-      const { group, application, user, place, pickup, issue } = context
+      const { group, application, user, place, activity, issue } = context
```

Comment:
Similar problem: context comes from API - maybe rewrite `pickup` to `activity` in the `update` mutation, then we don't need to care about it anymore in the remaining code?

Reply:
I fixed it in the dumb way, plus also fixing the enriched factories that were still creating contexts with an `activity`, not a `pickup`... it made the snapshots fit better too... 

I'm hoping the backend won't be too far behind...